app/forms/api/profile_creation_form.rb

```diff
@@ -32,7 +30,7 @@ def submit
         errors: errors,
         extra: extra_attributes,
       )
-      [response, personal_key]
+      [response, personal_key, gpo_code]
```

Comment:
this is a funky way to get around our "form objects always return form response" .... I would prefer that we jstu read `.personal_key` and `.gpo_code` off of the form object (assume they are set as side-effects of submit) rather than cram it into the response?

I don't feel strongly and it was already non-standard here, so no need to change if we don't want to

Reply:
Yeah initially I was thinking the same (set instance variable for gpo_code and read it from form), but I opted for this just to match consistency on how we were returning personal key.

Happy to make the change though.

Reply:
Updated in cac953fbc.